ts-client/src/vault/strategy/cypher.ts

```diff
@@ -0,0 +1,146 @@
+import { AccountMeta, Cluster, PublicKey, Transaction, TransactionInstruction } from '@solana/web3.js';
+import {
+  CypherClient,
+  Cluster as CypherCluster,
+  derivePublicClearingAddress,
+  deriveAccountAddress,
+  deriveSubAccountAddress,
+  derivePoolNodeAddress,
+  derivePoolNodeVaultAddress,
+  derivePoolNodeVaultSigner,
+} from '@mercurial-finance/cypher-client';
+import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
+import { TokenInfo } from '@solana/spl-token-registry';
+import BN from 'bn.js';
+
+import { SEEDS } from '../constants';
+import { AffiliateVaultProgram, VaultProgram } from '../types';
+import { Strategy, StrategyHandler } from '.';
+
+const PROGRAM_ID = 'CYPH3o83JX6jY6NkbproSpdmQ5VWJtxjfJ5P8veyYVu3';
```

Comment:
Can we get it from https://github.com/chugach-foundation/cypher-client-ts-v3/blob/main/src/constants/shared.ts#L36?